util/rate_limiter_test.cc

```diff
@@ -117,12 +119,26 @@ TEST_F(RateLimiterTest, Rate) {
               arg.request_size - 1, target / 1024, rate / 1024,
               elapsed / 1000000.0);
 
-      ASSERT_GE(rate / target, 0.80);
+      ++samples;
+      if (rate / target >= 0.80) {
+        ++samples_at_minimum;
+      }
       ASSERT_LE(rate / target, 1.25);
     }
   }
-}
+
+  // This can fail in heavily loaded CI environments
+  if (getenv("SANDCASTLE")
+#if (defined(TRAVIS) || defined(CIRCLECI)) && defined(OS_MACOSX)
```

Comment:
Out of curiosity, why is this skipped on MacOS?  One of these is not like the other...

Reply:
I'm not taking ownership of that. Just reducing the grossness of an existing hack job.

Reply:
Apparently does still fail in MacOS on CircleCI: Skipped minimum rate check (9 / 10 passed)